WordPress/Classes/ViewRelated/Post/WPPostViewController.m

```diff
@@ -556,6 +568,42 @@ - (void)configureMediaUpload
     self.mediaProgressView = [[UIProgressView alloc] initWithProgressViewStyle:UIProgressViewStyleBar];
 }
 
+- (void)createMediaThumbnailCallback:(NSURL *)thumbnailURL mediaID:(NSString *)mediaID isImage:(BOOL)isImage
+{
+    [[NSOperationQueue mainQueue] addOperationWithBlock:^{
+        if (isImage) {
+            [self.editorView insertLocalImage:thumbnailURL.path uniqueId:mediaID];
+        } else {
+            [self.editorView insertInProgressVideoWithID:mediaID usingPosterImage:thumbnailURL.path];
+        }
+    }];
+}
+
+- (void)completeCreateMedia:(Media *)media error:(NSError *)error mediaID:(NSString *)mediaID
```

Comment:
I'd perhaps rename this one too... `handleCreatedMedia:`? `handleNewMedia:`? Something like that?